desktop/esrWinFull/distribution/distribution.ini

```diff
@@ -11,12 +11,10 @@ app.distributor="mozillaonline"
 app.distributor.channel="esrWinFull"
 app.partner.mozillaonline="mozillaonline"
 app.chinaedition.channel="esr.firefox.com.cn"
-browser.newtabpage.activity-stream.default.sites=""
+browser.aboutwelcome.enabled=false
```

Comment:
Similarly, because the new about:welcome is not yet customizable.

See https://bugzil.la/1634005 and the same change was made to release channel in #12